### ISSUE TITLE
Arch Linux Support

### ISSUE THREAD

**Issue by Derrekito** [jdjaxon/linux_cac#24]
I retooled the script to support Arch Linux. Is there any interest in finding a add these changes? As of now, it replaces the current script -  so my fork has two independent scripts to support debian based distros or arch. See my fork: https://github.com/Derrekito/linux_cac_arch

**Comment by jdjaxon** [comment]
Of course! I want to add support for as many distros as possible. I will look over your fork and see if I can find a way to integrate it into this. I appreciate the feedback and that you took the time to solve that. I will close this issue once we have support for Arch.

**Comment by Derrekito** [comment]
I'm happy this is well received. I went ahead and merged the scripts. For easy comparison there's now three scripts, the merged version as well as the debian and arch versions. I'm happy to keep working towards a pull request.
